import-export-cli/cmd/deprecated/addApi.go

```diff
@@ -203,7 +203,7 @@ func createAPI(configMapNames []string, timestamp string, balInterceptors []stri
 	//assigning values to API cr
 	apiCrd.Name = flagApiName
 	apiCrd.Namespace = flagNamespace
-	apiCrd.Spec.Definition.SwaggerConfigmapNames = configMapNames
+	//apiCrd.Spec.Definition.SwaggerConfigmapNames = configMapNames
```

Comment:
Shall we remove this commented code segment?

Reply:
Fixed with 60ffa8adb24316d4d7b9fcbdc61476b31af7f787

Reply:
Fixed with e13921e5fc8c8e68de28237ef3e0fcaef1a51f9b